scripts/image_dl_test_deploy.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
 # Download cloud image, push it to glance and create test instance
 
-if [[ $(dirname $0) == *epouta* ]]; then
-source $(dirname $0)/image_epouta_constants.sh
+echo "=== "$(date)
+
+# Expiting the file location to be:
+# /SOME/PATH/diskimage-builder-scripts.${openstack_environment}/image_${openstack_environment}_constants.sh
+openstack_environment="$( echo $(dirname $0) | sed 's/^.*diskimage-builder-scripts\.//g' |cut -d '/' -f 1)"
```

Comment:
Very nice. Seems to work when running. Ran the line in the cronjob manually from $HOME of imgbuilder. Also adjusted the crontab and let a centos7 build run and that worked too.